aem-authoring-toolkit-api/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/anchorbutton/AnchorButton.java

```diff
@@ -0,0 +1,132 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.api.annotations.widgets.anchorbutton;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.exadel.aem.toolkit.api.annotations.meta.EnumValue;
+import com.exadel.aem.toolkit.api.annotations.meta.PropertyMapping;
+import com.exadel.aem.toolkit.api.annotations.meta.PropertyRendering;
+import com.exadel.aem.toolkit.api.annotations.meta.ResourceType;
+import com.exadel.aem.toolkit.api.annotations.meta.ResourceTypes;
+import com.exadel.aem.toolkit.api.annotations.widgets.common.ElementVariantConstants;
+import com.exadel.aem.toolkit.api.annotations.widgets.hyperlink.HyperlinkIconSize;
+import com.exadel.aem.toolkit.api.annotations.widgets.hyperlink.HyperlinkLinkchecker;
+
+/**
+ * Used to set up
+ * https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/granite-ui/api/jcr_root/libs/granite/ui/components/coral/foundation/anchorbutton/index.html
+ * AnchorButton is a component to represent a standard HTML hyperlink (<a>), but to look like a button in TouchUI dialog
+ */
+@Target(ElementType.FIELD)
+@Retention(RetentionPolicy.RUNTIME)
+@ResourceType(ResourceTypes.ANCHOR_BUTTON)
+@PropertyMapping
+@SuppressWarnings("unused")
+public @interface AnchorButton {
+    /**
+     * Maps to the href attribute of this TouchUI dialog component's node
+     * @return String value
+     */
+    String href();
+
+    /**
+     * Maps to the href attribute of this TouchUI dialog component's node.
+     * This is usually used to produce different value based on locale
+     * @return String value
+     */
+    String hrefI18n() default "";
+
+    /**
+     * Maps to the body text of the element of this TouchUI dialog component's node
+     * @return String value
+     */
+    String text();
+
+    /**
+     * Maps to the target attribute of this TouchUI dialog component's node
+     * @return String value
+     */
+    String target() default "";
+
+    /**
+     * Visually hide the text. It is RECOMMENDED that every button has a text for a11y purpose.
+     * Use this property to hide it visually, while still making it available for a11y
+     * @return True or false
+     */
+    @PropertyRendering(ignoreValues = "false")
+    boolean hideText() default false;
+
+    /**
+     * Maps to the x-cq-linkchecker attribute of this TouchUI dialog component's node
+     * @return One of {@code HyperlinkLinkchecker} values
+     * @see HyperlinkLinkchecker
+     */
+    @PropertyRendering(ignoreValues = "NONE")
+    HyperlinkLinkchecker xCqLinkchecker() default HyperlinkLinkchecker.NONE;
```

Comment:
this enum us used in more than one widget. Perhaps it makes sense to mode it to some common package (`com.exadel.aem.toolkit.api.annotations.widgets.common` seems to be a good candidate)